libs/common/RespWriteUtils.cs

```diff
@@ -457,5 +441,23 @@ public static bool WriteArrayWithNullElements(int len, ref byte* curr, byte* end
             }
             return true;
         }
+
+        /// <summary>
+        /// Write newline (\r\n) to <paramref name="curr"/>
+        /// </summary>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static void WriteNewline(ref byte* curr) => WriteBytes<ushort>(ref curr, "\r\n"u8);
+
+        /// <summary>
+        /// Writes two bytes to <paramref name="curr"/>
```

Comment:
Gonna wait for CI to finish before fixing this comment:

```suggestion
        /// Write <paramref name="bytes"/> to <paramref name="curr"/> as type <typeparamref name="T"/> sized value.
```